packages/Manager/.batect/database/init.sql

```diff
@@ -15,9 +15,10 @@ CREATE EXTENSION postgis;
 REVOKE ALL PRIVILEGES ON SCHEMA public FROM PUBLIC;
 
 CREATE USER eop_manager_migrations_user WITH PASSWORD 'password' NOINHERIT;
-
+CREATE SCHEMA plan_limits;
```

Comment:
Do we need this now that we're continuing to use use the default schema?

Reply:
Good catch, cheers